README.md

```diff
@@ -88,11 +88,14 @@ use the `artifact-name` parameter:
 The main [SBOM action](action.yml), responsible for generating SBOMs
 and attaching them to your wofklow and releases.
 
-| Parameter       | Description                                                                                             | Default                     |
-| --------------- | ------------------------------------------------------------------------------------------------------- | --------------------------- |
-| `path`          | A path on the filesystem to scan. This is mutually exclusive to `image`.                                | \<current directory>        |
-| `image`         | A container image to scan. This is mutually exclusive to `path`.                                        |
-| `artifact-name` | The name to use for the generated SBOM artifact. See: [Naming the SBOM output](#naming-the-sbom-output) | `sbom-<job>-<step-id>.spdx` |
+| Parameter           | Description                                                                                             | Default                     |
+| ------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------- |
+| `path`              | A path on the filesystem to scan. This is mutually exclusive to `image`.                                | \<current directory>        |
+| `image`             | A container image to scan. This is mutually exclusive to `path`.                                        |
+| `registry`          | The container registry to use                                                                           |
```

Comment:
I think it would help users if we were explicit about what kind of value we're asking for here.

For example:
- Is this a name of a registry from an enumerated list? E.g. `"dockerhub"`
- Is this a domain name? E.g. `"ghcr.io"`
- Is this full URL, to be used a prefix to image repository paths? E.g. `"https://index.docker.io"`

Reply:
Should we default to any registry in particular? Docker Hub? GHCR?

Reply:
My thoughts here are: we do not default to a registry. This allows us to decouple the `image` and `registry` configs, if a user adds an `image` with no registry, it would try to use the local docker daemon. If the user adds a registry, it would try to use that, keeping the Syft command line parameters hidden. Also I would think it would be a domain name e.g. `ghcr.io` but if the user wanted to use http, they could specify `http://...` etc, and we map those choices to the appropriate syft command line config

Reply:
Ok, made a few changes based on this feedback, please let me know if things make more sense now!